template/docroot/sites/default/settings/base.settings.php

```diff
@@ -27,7 +27,7 @@
 $is_ah_prod_env = ($ah_env == 'prod' || $ah_env == '01live');
 $is_ah_stage_env = ($ah_env == 'test' || $ah_env == '01test');
```

Comment:
There are some ACE sites the use `stg` for stage instead of `test`. Do we want to account for these. Not sure how many in number overall.